__tests__/__snapshots__/index.spec.js.snap

```diff
@@ -47,6 +47,7 @@ exports[`toMatchImageSnapshot passes diffImageToSnapshot everything it needs to
   "diffDirection": "horizontal",
   "failureThreshold": 0,
   "failureThresholdType": "pixel",
+  "onlyDiff": undefined,
```

Comment:
the docs mention the default is `false` but this snapshot sees `undefined`, should it not see `false`?

Reply:
I wanted to follow some other defaults, that are set to undefined, but mention in the docs that by default it's considered as false (both false and undefined behave the same way)

I can update the default value so that the 2 would match 👍